test/__tests__/ts-decorator-auto-accessor-test.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+jest.autoMockOff();
+const defineTest = require("../../src/testUtils").defineTest;
+
+describe("should be parse typescript decoratorAutoAccessors correctly", function () {
+  defineTest(__dirname, "ts-decorator-auto-accessor", null, null, {
```

Comment:
Please use `defineInlineTest`. You can also define the transform inline if you use `defineInlineTest`, e.g.

```
function transformer(file, api) {
  .....
}

defineInlineTest(transformer, .......)
```

Reply:
Thanks for review. I switched to inline test.